app/src/main/java/org/mozilla/fenix/compose/TabSubtitle.kt

```diff
@@ -26,18 +28,20 @@ import org.mozilla.fenix.theme.Theme
 @Composable
 fun TabSubtitle(
     text: String,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    fontSize: TextUnit = 12.sp,
+    color: Color = when (isSystemInDarkTheme()) {
+        true -> FirefoxTheme.colors.textPrimary
+        false -> FirefoxTheme.colors.textSecondary
+    }
```

Comment:
I am actually looking to replace these with `PrimaryText()` and `SecondaryText()`. Can you look to use either one of those in your implementation?